docker/README.md

```diff
@@ -80,6 +80,64 @@ Go to https://localhost/api_keys, generate key and secret. Then run:
     # Where KEY is your openQA instance key
     # and SECRET is your openQA instance secret
 
+## Alternative: Run the Data & Web UI containers in HA mode with docker-compose
+
+    # To create the containers
+    # in the directory openQA/docker/webui execute: 
+    docker-compose up -d
+
+### Change the exported port for the load balancer
+
+By default the load balancer exposes the web UI in the port 9526. That can be
+changed in the docker-compose.yaml for the service haproxy. For instance to
+expose as port 80, change the first number in the tuple of ports for this service
+
+```
+ports:
+  - "80:9526"
+```
+
+### Change the number of replicas for the web uUI service
+
+You can change it in two steps:
+
+1. In the docker-compose.yaml change the value for replicas in the deploy section for the web UI   service
+2. Add more lines like 'server webui2 webui_webui_2:9526' to the haproxy.cfg file, setting the number of each replica. For instance for 4 replicas use:
```

Comment:
Please configure your editor to break lines after a sensible limit. I understand that so far the file is already quite inconsistent but we have to start somewhere :)